src/PrintBox_text.ml

```diff
@@ -56,6 +97,7 @@ end = struct
     | Char of char
     | String of string
     | Str_slice of string * int * int
+    | Str_slice_bracket of string * string * int * int * string
```

Comment:
Maybe use an inline record to distinguish all the integers and strings in there ?